helix-view/src/info.rs

```diff
@@ -0,0 +1,54 @@
+use crate::input::KeyEvent;
+use std::fmt::Write;
+use unicode_width::UnicodeWidthStr;
+
+#[derive(Debug)]
+/// Info box used in editor. Rendering logic will be in other crate.
+pub struct Info {
+    /// Title kept as static str for now.
+    pub title: &'static str,
+    /// Text body, should contains newline.
+    pub text: String,
```

Comment:
Not sure if keeping the text as `String` is the best idea. It needs to use `lines()` to render later.